docs/aws.md

```diff
@@ -540,6 +540,7 @@ and the Batch Execution Role.
 
 :::{note}
 This feature requires the use {ref}`Wave <wave-page>` container provisioning service.
+This is because `s5cmd` is used to stage and unstage data which is augmented into the task container via wave.
```

Comment:
```suggestion
This feature requires the {ref}`Wave <wave-page>` container provisioning service.
This is because `s5cmd` is used to stage and unstage data that is augmented into the task container via wave.
```

Reply:
I'd improve like this 

> :::{note}
Under the hood Nextflow uses the [s5cmd](https://github.com/peak/s5cmd) to download the task input data
and upload the task outputs. To allow this capability your pipeline need to enable the Wave service 
in the Nextflow configuration as shown in the above example. See {ref}`Wave <wave-page>` documentation for more details.
:::